src/core/sources/RTMP.ts

```diff
@@ -151,7 +151,6 @@ const connectDevice = async (id: string) => {
         autoGainControl: false,
         channelCount: 2,
         echoCancellation: false,
-        latency: 0,
```

Comment:
You may keep this property as it is. Although TypeScript 5.4-beta has removed this property from its typing, it is still supported according to web API standards. You can refer to the documentation at: https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints/latency

Reply:
interesting - I'll ts-ignore the line and link to this